jobs/config.json

```diff
@@ -3233,6 +3233,30 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-e2e-gce-scale-coredns": {
+    "args": [
+      "--cluster=gce-scale-cluster",
+      "--env=CLUSTER_DNS_CORE_DNS=true",
+      "--env=HEAPSTER_MACHINE_TYPE=n1-standard-8",
+      "--env-file=jobs/env/ci-kubernetes-e2e-scalability-common.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-large-performance.env",
+      "--extract=ci/latest",
+      "--gcp-master-image=gci",
+      "--gcp-node-image=gci",
+      "--gcp-nodes=3",
+      "--gcp-project=kubernetes-scale",
```

Comment:
We don't want to run it in this project (let's remove this line and rely on test-infra tooling to choose the correct one).

Reply:
+1. It's too small and can fit in regular testinfra projects - we don't need a large one.